Dockerfile

```diff
@@ -40,6 +40,11 @@ ENV TIMEZONE=Europe/Berlin \
     BACKUP_ENABLED=true \
     BACKUP_CRON_EXPRESSION="0 * * * *"
 
+USER root
+
+RUN mkdir /palworld
+RUN chown 1000:1000 /palworld
```

Comment:
This should use the PGID/PUID variables so it can be customized

Reply:
Given that this runs on the build step, it wouldn't really be configurable when someone just uses the image as is; i'll update it to `steam:steam` as that's makes it a little bit clearer who should be owning that directory (which is the user running the installation)